src/python/py/models/builder.py

```diff
@@ -1338,11 +1349,10 @@ def make_attention_mask_reformatting_for_mha(self):
         end_add_shape = ["batch_size", 1, "source_sequence_length", "target_sequence_length"]
         self.make_add(end_add_name, end_add_inputs, dtype=self.io_dtype, shape=end_add_shape) # Shape of mask is now (B, 1, S, T)
 
-        # TODO: replace Concat with Expand for performance gains
         concat_name = f"{basename}/Concat"
```

Comment:
If `Tile` is [good to use](https://github.com/microsoft/onnxruntime-genai/pull/426/files#r1596059754), then can this be updated to the following?

```suggestion
        tile_name = f"{basename}/Tile"
```

If you can make similar changes to replace `concat` with `tile` in the below lines, that would be helpful.

Reply:
done